validator-auction-rest-server/README.md

```diff
@@ -30,6 +30,12 @@ ETH_EVENTS_TOKEN=1d59aa2e-2036-46fd-b54e-f568db07dea1
 ```
 The eth.events access token used to send the requests.
 
+```
+START_TIMESTAMP=1561107511
```

Comment:
I think we should use a name that is more unlikely to be used by something else
`VALIDATOR_AUCTION_START_TIMESTAMP`

Reply:
Agree.